src/qililab/platform/platform.py

```diff
@@ -714,7 +713,7 @@ def execute(
         repetition_duration: int,
         num_bins: int = 1,
         queue: Queue | None = None,
-    ) -> Result:
+    ) -> Result | QbloxResult:
```

Comment:
Why do we change this?

Reply:
this is to merge changes from `main` which are not in this branch, so it's not a change from `new-qblox-firmware`. This change was introduced to support post-processing results in platform when we re-order them at `self._order_result`

Reply:
I am confused. main branch does not contain `QbloxResult`. 

![image](https://github.com/qilimanjaro-tech/qililab/assets/3442573/b59937b7-fc98-4708-b65c-9e9dc599cc08)

Reply:
ok apparently I was wrong, these changes are from https://github.com/qilimanjaro-tech/qililab/pull/656, which we did not implement in `main`